pandas-stubs/core/indexes/accessors.pyi

```diff
@@ -140,7 +140,6 @@ class _DatetimeOtherOps(Generic[_DTOtherOpsDateReturnType, _DTOtherOpsTimeReturn
     @property
     def timetz(self) -> _DTOtherOpsTimeReturnType: ...
 
-class DatetimeAndPeriodProperties(_DatetimeFieldOps[Series[int]]): ...
```

Comment:
This isn't used anywhere.

Reply:
SEems like other changes made didn't delete it